cupy/logic/truth.py

```diff
@@ -57,3 +57,67 @@ def any(a, axis=None, out=None, keepdims=False):
 
     assert isinstance(a, cupy.ndarray)
     return a.any(axis=axis, out=out, keepdims=keepdims)
+
+
+def in1d(ar1, ar2, assume_unique=False, invert=False):
+    """Tests whether each element of a 1-D array is also present in a second
+    array.
+
+    Returns a boolean array the same length as ``ar1`` that is ``True``
+    where an element of ``ar1`` is in ``ar2`` and ``False`` otherwise.
+
+    Args:
+        ar1 (cupy.ndarray): Input array.
+        ar2 (cupy.ndarray): The values against which to test each value of
+            ``ar1``.
+        assume_unique (bool, optional): Ignored
+        invert (bool, optional): If ``True``, the values in the returned array
+            are inverted (that is, ``False`` where an element of ``ar1`` is in
+            ``ar2`` and ``True`` otherwise). Default is ``False``.
+
+    Returns:
+        cupy.ndarray, bool: The values ``ar1[in1d]`` are in ``ar2``.
+
+    """
+    # Ravel both arrays, behavior for the first array could be different
+    ar1 = ar1.ravel()
+    ar2 = ar2.ravel()
+    if ar1.size == 0 or ar2.size == 0:
+        if invert:
+            return cupy.ones(ar1.shape, dtype=cupy.bool_)
+        else:
+            return cupy.zeros(ar1.shape, dtype=cupy.bool_)
+
+    shape = (ar1.size, ar2.size)
+    ar1_broadcast = cupy.broadcast_to(ar1[..., cupy.newaxis], shape)
+    ar2_broadcast = cupy.broadcast_to(ar2, shape)
+    count = (ar1_broadcast == ar2_broadcast).sum(axis=1)
+    if invert:
+        return count == 0
+    else:
+        return count > 0
+
+
+def isin(element, test_elements, assume_unique=False, invert=False):
+    """Calculates element in ``test_elements``, broadcasting over ``element``
+    only. Returns a boolean array of the same shape as ``element`` that is
+    ``True`` where an element of ``element`` is in ``test_elements`` and
+    ``False`` otherwise.
+
+    Args:
+        element (cupy.ndarray): Input array.
+        test_elements (cupy.ndarray): The values against which to test each
+            `value of ``element``. This argument is flattened if it is an
```

Comment:
There's a superfluous symbol. (Please check the output of docs by yourself)
```suggestion
            value of ``element``. This argument is flattened if it is an
```